indra/newview/llgltfmateriallist.cpp

```diff
@@ -371,24 +371,19 @@ void LLGLTFMaterialList::queueApply(const LLViewerObject* obj, S32 side, const L
 {
     if (asset_id.isNull() || material_override == nullptr)
     {
-        queueApply(obj, side, asset_id);
-    }
-    else
-    {
-        LLGLTFMaterial* material = new LLGLTFMaterial(*material_override);
-        sApplyQueue.push_back({ obj->getID(), side, asset_id, material });
+        return;
```

Comment:
Please, consider adding llassert or LL_WARN here. Something along the lines of 'tried to use wrong queueApply variant to reset material'.